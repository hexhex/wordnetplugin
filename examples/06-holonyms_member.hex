%% this returns all (member) holonyms of a word
%% POS: nouns

%% the search number for (member) holonyms
%% is 6.
%% Note: In the WordNet documentation this is
%%       the search number for member meronyms.
%%       In fact search 6 fetches words, for
%%       which *word* is a meronym (and these
%%       are its holonyms)

word("lion").
holonyms(W,SW,H,SH) :- &wordnet[6,W](1,SW,H,SH),word(W).
