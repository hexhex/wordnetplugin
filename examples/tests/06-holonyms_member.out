{holonyms("lion",1,"pride",2), holonyms("lion",1,"genus_Panthera",1), holonyms("lion",1,"Panthera",1), word("lion")}
